Add make-docs script that can be used to build multiple documentation sets (#211)
grafana/writers-toolkit: docs/sources/writing-guide/tooling-and-workflows/run-a-local-webserver/index.md

---
title: Run a local documentation webserver with the make-docs script
menuTitle: Run a local documentation webserver

> description: How to run a local documentation webserver with the make-docs script.

Reviewer: ```suggestion
description: Run a local documentation web server with the make-docs script.
```

---

# Run a local documentation webserver with the make-docs script

> The [`make-docs`](https://github.com/grafana/writers-toolkit/blob/main/scripts/make-docs) script can be used to mount multiple documentation sets in a single build.
> It is used by the `make docs` Make target in project repositories and can also be run separately.

Reviewer: ```suggestion
To build multiple documentation sets within a single build, you can use the [`make-docs`](https://github.com/grafana/writers-toolkit/blob/main/scripts/make-docs) script, which you can also run on an individual documentation set.

> **Note:** The similar looking `make docs` Make target uses it within project repositories.
```

## Examples

The following examples assume:

1. Your working directory is the root of a project repository.
1. You have a copy of the `make-docs` script in that working directory.

> 1. The script is marked as executable.

Reviewer: ```suggestion
1. The script is executable.
```

> 1. You have a checkout of each of the project repositories used in the examples.

Reviewer: ```suggestion
1. You have a checkout of each of the project repositories that are used in the examples.
```
